### DISCUSSION TITLE
Ability to disable display of original drawing positions when using the Edit Shift command.

### DISCUSSION THREAD

**Discussion by beeheemooth** [opentoonz/opentoonz#6474] [Ideas]
When using Shift and Trace in Edit Shift mode, the original position of the drawings is displayed, which allows you to see the displacement of the drawings relative to each other and thus correctly place them one above the other. However, drawing becomes difficult at the same time, since the drawer sees twice as many lines and can easily get confused. It is especially difficult to draw when the displacement of the drawings is small and there are a lot of details - the lines form a continuous mess.
![Screen_02201](https://user-images.githubusercontent.com/25724133/117961414-c3cc3000-b326-11eb-84d5-fc5efb4deb9c.jpg)

A solution to this problem would be the ability to turn off the display of the original drawing positions when drawing, so that you can draw only in relation to offset drawings. 

Current
![Screen_0219](https://user-images.githubusercontent.com/25724133/117961495-da728700-b326-11eb-9480-19d7202a0fe3.jpg)

Expected
![Screen_02191](https://user-images.githubusercontent.com/25724133/117961510-de9ea480-b326-11eb-8bdb-371fffc7b9be.jpg)

**Comment by F-Burning** [comment]
Totally agree 💯

**Comment by gab3d** [comment]
seems useful 👍

**Comment by DarrenTAnims** [comment]
Absolutely. This would be soooo useful.

**Comment by KirumoKat** [comment]
I've been wanting to open such an issue for a long while since I started messing with OpenToonz myself, because I find this really annoying... Clip Studio Paint never had original frames in their actual position when I use shift and trace there, but in OpenToonz, it really becomes a mess, especially when shifted drawings are originally close to each other. Setting a lower paper opacity doesn't help because their stronger colors make it harder to draw. So yeah, I agree that this needs to be changed or at least have a toggle or something.
